pkg/aws/client.go

```diff
@@ -158,7 +159,7 @@ func (b *ClientBuilder) Build() (result Client, err error) {
 	}
 
 	// Create and populate the object:
-	result = &awsClient{
+	c := &awsClient{
```

Comment:
any reason to change the variable name?

Reply:
The return value result https://github.com/openshift/moactl/pull/65/files#diff-45cbc761a9f9e83f2c1f69c292f06703R99 has type `Client`, it is an interface. While `getClientDetails` only takes `*awsClient` type. So I cannot pass `result` to that function and have to rename the var here.

```
func getClientDetails(awsClient *awsClient)
```

Reply:
In that case you should probably make the return anonymous
```
func (b *ClientBuilder) Build() (Client, error)
```
Then you can keep
```
result := &awsClient{...}
```
Otherwise you have a named return that doesn't get used

Reply:
Yes, I think returning anonymous values is better. I will update this pr.

Reply:
Thanks, I think this works